README.arm.md

```diff
@@ -1,14 +1,24 @@
-# Julia binaries for ARM
+# Julia on ARM (Linux)
 
-[Nightly builds](https://status.julialang.org/download/linux-arm) are
-available for ARMv7-A.
+Julia support for ARM is work-in-progress, though good results have been
+reported for a number of commercially available devices. This file provides
+general guidelines for compilation on 32- and 64-bit variants of ARM
+processors, in addition to instructions for specific devices.
 
-# Hardware requirements
+A list of [known issues](https://github.com/JuliaLang/julia/labels/arm) on
+ARM is available.
+
+## 32-bit (ARMv6, ARMv7)
 
 Julia requires at least `armv6` and `vfpv2` instruction sets. It's recommended
 to use at least `armv7-a`. `armv5` or soft float are not supported.
 
-# Building Julia on ARM
+### Binaries
+
+[Nightly builds](https://status.julialang.org/download/linux-arm) are
+available for ARMv7-A.
+
+### Building Julia
 
 Julia has been compiled on several ARMv7 / Cortex A15 Samsung
 Chromebooks running Ubuntu Linux under Crouton, Raspberry Pi systems
```

Comment:
Since you are doing a overhaul of this file, we should update this too. No enabled tests should be failing (on armv7) (profile tests are disabled), backtraces are available.